nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -47,6 +47,10 @@ class BtActionNode : public BT::ActionNodeBase
   : BT::ActionNodeBase(xml_tag_name, conf), action_name_(action_name)
   {
```

Comment:
A note in the migration guide would be great that the BT nodes now have separate executors to ensure deterministic spinning of only the relevant data sources to each BT node